api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java

```diff
@@ -278,7 +278,7 @@ public Obs voidObs(Obs obs, String reason) throws APIException {
 	 * @throws APIException
 	 */
 	public Obs unvoidObs(Obs obs) throws APIException {
-		return dao.saveObs(obs);
+		return Context.getObsService().saveObs(obs,"unvoid obs");
```

Comment:
We might want to have an exception for Obs, there is some complex logic around updating Obs and this may introduce some obscure behavior.